src/base/memory_monitor_sr.cpp

```diff
@@ -20,18 +20,62 @@
  * memory_monitor_sr.cpp - Implementation of memory monitor module
  */
 
+#include <malloc.h>
+#include <cassert>
 #include <cstring>
 #include <algorithm>
-#include <cassert>
 
 #include "memory_monitor_sr.hpp"
 
+#ifndef HAVE_USR_INCLUDE_MALLOC_H
+#define HAVE_USR_INCLUDE_MALLOC_H
+#endif
+
+typedef struct mmon_metainfo
+{
+  uint64_t size;
+  int tag_id;
+  int magic_number;
+} MMON_METAINFO;
```

Comment:
I recommand this style for the readability.

```suggestion
typedef struct mmon_metainfo MMON_METAINFO;
typedef struct mmon_metainfo
{
  uint64_t size;
  int tag_id;
  int magic_number;
};
```

Reply:
What is the meaning of `tag_` in this context? How about `mem_id` or `tracked_id` if there is no special meaning of `tag_`?

Reply:
Done.

Reply:
http://jira.cubrid.org/browse/CBRD-25196
I intended to show the relation tag_name(like "base/heap_file.c:496") <-> tag_id <-> memory usage.